### ISSUE TITLE
can't browse local data from opencga-light: XMLHttpRequest cannot load local data - No 'Access-Control-Allow-Origin' header is present on the requested resource

### ISSUE THREAD

**Issue by mbsimonovic** [opencb/genome-maps#19]
not sure if this is genomemaps.org or opencga-light, but when opencga is started locally (v0.1.0), genomemaps.org can browse the folders but fails when BAM file as added as a track with the following error:

```
XMLHttpRequest cannot load http://localhost:61976/opencga/rest/storage/fetch?filepath=...GBR.low_coverage.20101123.bam&region=13:32885000-32894999&_=1440681422096. 
No 'Access-Control-Allow-Origin' header is present on the requested resource. 
Origin 'http://www.genomemaps.org' is therefore not allowed access. 
The response had HTTP status code 500.
```

**Comment by imedina** [comment]
Thank you very much for reporting and sorry for the inconveniences. New Genome Maps (GM) version v3.2 is being developed in parallel with OpenCGA https://github.com/opencb/opencga
OpenCGA is heading to v1.0 and we are making data formats, APIs and REST web services stable (https://github.com/opencb/opencga/wiki/RESTful-Web-Services). Current GM v3.2 can consume data from OpenCGA v0.6+ (Documentation will be updated pretty soon)
OpenCGA Light server functionality is also being taken care of, but until it is fully functional a complete OpenCGA server is needed. BTW, old OpenCGA Light functionality will be replaced by OpenCGA Storage micro-services at some point in current OpenCGA v0.7 release.

**Comment by mbsimonovic** [comment]
yeah, i also tried openCGA but it also doesn't show uploaded data as you can see on this screenshot:

![screen shot 2015-08-27 at 15 58 58](https://cloud.githubusercontent.com/assets/888008/9599258/b7813144-5092-11e5-9159-bb11c10373c8.png)
